cfgov/unprocessed/css/enhancements/base.less

```diff
@@ -13,3 +13,6 @@ nav ol ol {
 }
 
 // END enhancement for normalize-legacy-addon
+
+
+
```

Comment:
Could we remove these added blank lines, please?

Reply:
Removed in https://github.com/cfpb/cfgov-refresh/pull/4004/commits/4cfe69b11d39709735b32054a8de723be1883caa